packages/notifi-dapp-example/src/app/globals.css

```diff
@@ -0,0 +1,72 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
+@layer base {
+  :root {
+    /* Notifi global color */
```

Comment:
We still use css global variables to manage color.
This way will be able to have dark/mode switch if needed.